src/api/routes.ts

```diff
@@ -89,18 +91,34 @@ export function getPackage(req: auth.AuthRequest, res: express.Response): void |
 
       res.status(200).json({ name: pkgName, maintainers: users });
     } else {
-      return res.status(404).json('');
+      let pkgJsonPath = getFilePath(`packages/${pkgName}/package.json`);
+      if (pkgName.indexOf('@') !== -1) {
+        pkgName = pkgName.replace(/^(@.*)(\/)(.*)$/g, '$1%2F$3');
+      }
+
+      if (config.useUpstream) {
```

Comment:
this if is exactly the same as if starts on line 130. I think we could extract that in a function. I don't like code duplication... What do you think?